src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -688,8 +688,8 @@ class JsonPrinter : public Printer {
         BSLS_KEYWORD_OVERRIDE
     {
         closeBraceIfOpen();
-        d_ostream << "  \"JournalOperationsNumber\": \""
-                  << journalOpRecordsCount << "\"";
+        d_ostream << "  \"JournalOperationsNumber\": "
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.